Sharprompt/Prompt.Bind.cs

```diff
@@ -59,14 +59,14 @@ private static bool MakeConfirm(PropertyMetadata propertyMetadata)
 
     private static T MakeInputCore<T>(PropertyMetadata propertyMetadata)
     {
-        return Input<T>(options =>
-        {
-            options.Message = propertyMetadata.Message;
-            options.DefaultValue = propertyMetadata.DefaultValue;
-            options.Placeholder = propertyMetadata.Placeholder;
-
-            options.Validators.Merge(propertyMetadata.Validators);
-        });
+        return propertyMetadata.AutoGenerated ? default :
```

Comment:
I believe the AutoGenerated determination can be made within the same method as the CanWrite modification. I would like to know the intent of adding it here.

Reply:
It was not made there because at that point the "EnsureOptions" method were not executed yet and you can not guarantee the presence of DisplayAttribute, it would be necessary some validation steps and I thought it looked more clear been placed where the others DisplayAttribute properties are

Reply:
Very good point, the changes here were undone and the solution moved into the same method as the CanWrite modification.